examples/quickstart/jupyter-notebooks/druid-visualization.ipynb

```diff
@@ -0,0 +1,623 @@
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "b5364774-b8e7-46a0-82d3-f1a9f09a6cc0",
+   "metadata": {},
+   "source": [
+    "Now, configure the y-axis:\n",
+    " - Set the minimum value to `0`.\n",
+    " - Set the visible range to `(0,40)`."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "c5f83f5c-f546-4a71-be36-30f659a9b94b",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "bar_plot.y_range.start = 0\n",
+    "y_range=(0, 40)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "8b54c25a-07af-4ade-96d5-15ca04178fe2",
+   "metadata": {},
+   "source": [
+    "Finally, display your plot with the `show()` method."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "44a604ec-208c-4d7a-9d25-97b308328407",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "show(bar_plot)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "772a20f2-2ea9-4e67-96fa-0a1eab32bb5a",
+   "metadata": {},
+   "source": [
+    "## Display data with a line graph"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "9fd43bc4-d048-4319-8dd0-2b0ca94fd33b",
+   "metadata": {},
+   "source": [
+    "In this section, you'll create a line graph that compares the following, per channel:\n",
+    "\n",
+    "* Total additions\n",
+    "* Number of unique editors\n",
+    "* Number of bot edits"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "1f915e03-103c-4151-93c4-ad779741f2a5",
+   "metadata": {},
+   "source": [
+    "First, change the query to include the editors and robots and load the results into a new pandas object."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "910920f5-db6a-4c3a-95fc-d25763d45de4",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "query = \"\"\"\n",
+    "        SELECT channel, SUM(added) AS additions,\n",
+    "        COUNT (DISTINCT user) as editors,\n",
+    "        SUM(CASE WHEN isRobot='true' THEN 1 ELSE 0 END) AS robots\n",
+    "        FROM wikipedia_api\n",
+    "        GROUP BY channel ORDER BY additions DESC LIMIT 10\n",
+    "        \"\"\"\n",
+    "\n",
+    "# Update the payload.\n",
+    "payload = json.dumps({\n",
+    "  \"query\" : query\n",
+    "})\n",
+    "\n",
+    "# Submit the request.\n",
+    "response = requests.request(http_method, druid_host+endpoint, headers=headers, data=payload)\n",
+    "\n",
+    "# Add the results to a pandas DataFrame.\n",
+    "editors_pd = pd.read_json(json.dumps(response.json()), orient='records')\n",
+    "\n",
+    "editors_pd"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "5f761a0b-efe1-4e4a-bfa5-a80aca3e2cc8",
+   "metadata": {},
+   "source": [
+    "Set up values for your plots:\n",
+    "- Channels remains the basis for the x-axis.\n",
+    "- This time, change the resolution for additions to 1000 so it will fit nicely on our plot with the editors and the robots values.\n",
```

Comment:
```suggestion
    "- This time, change the resolution for additions to 1000, so it will fit nicely on our plot with the editors and the robots values.\n",
```